doc/source/PreConfiguredSites.rst

```diff
@@ -521,9 +521,10 @@ The following is required for building new spack environments and for using spac
 .. code-block:: console
 
    module purge
-   module use /scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles
-   module load miniconda/3.9.12
-   module load ecflow/5.5.3
+   ### module use /scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles
```

Comment:
@RatkoVasic-NOAA Can you please update this?

Reply:
I was just about to :-)